pykafka/balancedconsumer.py

```diff
@@ -124,6 +125,9 @@ def __init__(self,
             server should return for a fetch request. If insufficient data is
             available, the request will block until sufficient data is available.
         :type fetch_min_bytes: int
+        :param fetch_error_backoff_ms: *UNUSED*.
```

Comment:
@johnistan Do you have an opinion on this? On one hand, there's a benefit to keeping the interfaces to the python and rdkafka consumers in sync. On the other, it's very confusing to first-time users of this API to have a kwarg that doesn't actually do anything.